config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java

```diff
@@ -855,6 +855,17 @@ private static void addSearchNodeExecutorMetrics(Set<Metric> metrics, String pre
         metrics.add(new Metric("vds.distributor.visitor.sum.latency.average")); // TODO: Remove in Vespa 8
         metrics.add(new Metric("vds.distributor.visitor.sum.ok.rate"));
         metrics.add(new Metric("vds.distributor.visitor.sum.failures.total.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.notready.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.notconnected.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.wrongdistributor.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.safe_time_not_reached.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.storagefailure.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.timeout.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.busy.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.inconsistent_bucket.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.notfound.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.concurrent_mutations.rate"));
+        metrics.add(new Metric("vds.distributor.visitor.sum.failures.test_and_set_failed.rate"));
```

Comment:
`concurrent_mutations` and `test_and_set_failed` metrics do not need to be included, as they cannot happen for read-only operations